mage/main.go

```diff
@@ -255,7 +237,7 @@ func run(cmd string, args ...string) int {
 	c.Stdout = os.Stdout
 	c.Env = os.Environ()
 	if verbose {
-		c.Env = append(c.Env, "MAGEFILE_VERBOSE=1")
+		c.Env = append(c.Env, "MAGEFILE_VERVOSE=1")
```

Comment:
Typo introduced here, VERBOSE to VER_V_OSE?

Reply:
Yep, thanks!  I added a test for that, too, so next time if I screw it up, at least a test will fail :)